src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/Main.java

```diff
@@ -126,13 +133,22 @@ private static void processFile(final File inputFile, final File outputFile) thr
                     "that was submitted to APIView was named " + inputFile.getName()));
         }
 
-        // Write out to the filesystem
-        try (JsonWriter jsonWriter = JsonProviders.createWriter(Files.newBufferedWriter(outputFile.toPath()))) {
-            apiListing.toJson(jsonWriter);
+        try {
+            // Write out to the filesystem, make the file if it doesn't exist
+            if (!outputFile.exists()) {
+                if (!outputFile.createNewFile()) {
+                    System.out.printf("Failed to create output file %s%n", outputFile);
```

Comment:
Same here - exit if file cannot be created.

Reply:
in this case I think we keep going - just because we support multiple files, and the others might work.